server/services/store/sqlstore/blocks.go

```diff
@@ -337,7 +326,27 @@ func (s *SQLStore) InsertBlock(c store.Container, block model.Block) error {
 		block.DeleteAt,
 	)
 
-	_, err = query.Exec()
+	// TODO: migrate this delete/insert to an upsert
+	deleteQuery := s.getQueryBuilder().Delete(s.tablePrefix + "blocks").Where(sq.Eq{"id": block.ID})
+	_, err = sq.ExecContextWith(ctx, tx, deleteQuery)
+	if err != nil {
+		tx.Rollback()
+		return err
+	}
+
+	_, err = sq.ExecContextWith(ctx, tx, query.Into(s.tablePrefix+"blocks"))
```

Comment:
Minor: spacing around + operator. Not sure if Go lint would catch this anyway.

Reply:
Yes, weird, I would expect that to be autoformatted.

Reply:
Looks like the auto formatting of go is removing the spaces, so I think we should ignore this case and keep it as is now.